starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -49,4 +49,18 @@ public void findAll() {
     List<Inventory> actualInventory = this.inventoryDAO.findAll();
     Assert.assertFalse(actualInventory.isEmpty());
   }
+
+  /**
+   * Test Create method.
+   */
+  @Test
+  public void createInsertsInventory() {
+    List<Inventory> existingInventory = this.mongoTemplate.findAll(Inventory.class);
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
+    inventory.setProductType(PRODUCT_TYPE);
+    this.inventoryDAO.create(inventory);
+    List<Inventory> inventoryPostInsertion = this.mongoTemplate.findAll(Inventory.class);
+    Assert.assertTrue(inventoryPostInsertion.size() == existingInventory.size() + 1);
```

Comment:
How do we know the inventory you wanted to create is actually the inventory that is added to the list?

Reply:
added additional assertions to ensure that the Inventory object created in the test is actually the one that gets inserted

Reply:
Great job with the extra steps to verify it was properly saved. One small thing: when you're dealing with items in a database collection, generally if the id matches what you expect it to be, the other fields will match (unless you are updating/editing). For the purposes of this specific test, we are testing create, so we don't need to check the other fields (name and product type) since the ids are unique, if the id of the created inventory exists, it is usually safe to assume that the creation of the object was successful.